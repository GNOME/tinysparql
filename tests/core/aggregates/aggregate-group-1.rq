PREFIX : <http://example/> 

SELECT ?a COUNT(DISTINCT ?b) SUM(?ib) GROUP_CONCAT(?sb, "|")
{ ?a a :A ;
     :b ?b .
  ?b :ib ?ib .
  ?b :sb ?sb .
  FILTER (?ib>2) }
GROUP BY ?a
