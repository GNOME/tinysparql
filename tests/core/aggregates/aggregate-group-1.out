"http://example/x"	"2"	"8"	"string t|string y"
"http://example/z"	"1"	"13"	"string u"
